src/k8s-frameworkcontroller/deploy/k8s-frameworkcontroller.json.template

```diff
@@ -0,0 +1,37 @@
+{
+  "apiVersion": "apps/v1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "k8s-frameworkcontroller-sts"
```

Comment:
Why not use yaml

Reply:
I remembered you said current pai k8s cannot received fc yaml correctly, it's ok to change to yaml

Reply:
In v1.14.2 version k8s, it is fine to use yaml. Does pure k8s pai already use  v1.14.2?

Reply:
seems deployment code will use the version [here](https://storage.googleapis.com/kubernetes-release/release/stable.txt), changed to yaml